compose/ui/ui/src/skikoMain/kotlin/androidx/compose/ui/scene/ComposeSceneLayer.skiko.kt

```diff
@@ -190,3 +192,17 @@ internal fun rememberComposeSceneLayer(
     }
     return layer
 }
+
+/**
+ * Sets the content of the layer to [content].
+ */
+@Composable
+internal fun ComposeSceneLayer.applyContent(content: @Composable () -> Unit) {
```

Comment:
```suggestion
internal fun ComposeSceneLayer.SetContentEffect(content: @Composable () -> Unit) {
```
By the guidelines, the name should start with upper case and be a noun

Reply:
What guideline? This function used to be called `rememberLayerContent`, and all that has changed is its implementation.

Reply:
Ok, it makes sense according to the Compose API guidelines. I went with simply `ComposeSceneLayer.Content`, to avoid exposing implementation details in the name.